docs/querying/dimensionspecs.md

```diff
@@ -65,32 +65,35 @@ Returns dimension values transformed using the given [extraction function](#extr
 }
 ```
 
-`outputType` may also be specified in an ExtractionDimensionSpec to apply type conversion to results before merging. If left unspecified, the `outputType` defaults to STRING.
+`outputType` may also be specified in an `ExtractionDimensionSpec` to apply type conversion to results before merging. If left unspecified, the `outputType` defaults to STRING.
 
 Please refer to the [Output Types](#output-types) section for more details.
 
 ### Filtered DimensionSpecs
 
-These are only useful for multi-value dimensions. If you have a row in Apache Druid that has a multi-value dimension with values ["v1", "v2", "v3"] and you send a groupBy/topN query grouping by that dimension with [query filter](filters.md) for value "v1". In the response you will get 3 rows containing "v1", "v2" and "v3". This behavior might be unintuitive for some use cases.
+A filtered `DimensionSpec` is only useful for multi-value dimensions. Say you have a row in Apache Druid that has a multi-value dimension with values ["v1", "v2", "v3"] and you send a groupBy/topN query grouping by that dimension with a [query filter](filters.md) for a value of "v1". In the response you will get 3 rows containing "v1", "v2" and "v3". This behavior might be unintuitive for some use cases.
+
+This happens because "query filter" is internally used on the bitmaps and only used to match the row to be included in the query result processing. With multi-value dimensions, "query filter" behaves like a contains check, which will match the row with dimension value ["v1", "v2", "v3"]. 
```

Comment:
```suggestion
This happens because Druid uses the "query filter" internally used on the bitmaps only to match the row to include in query result processing. With multi-value dimensions, "query filter" behaves like a contains check that matches the row with dimension value ["v1", "v2", "v3"]. 
```
make sure I didn't mangle the meaning here. This whole bit on filtered dimensions is a little confusing.

Reply:
I think the final clause is non-defining, so it should still be ", which matches ..."  The first sentence is confusing but I think it could accurately say, "This happens because Druid uses the "query filter" internally on bitmaps only to match the row...." Will edit separately